utils/i2csfp/Makefile

```
@@ -0,0 +1,39 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=i2csfp
+PKG_VERSION:=20240616
+PKG_RELEASE:=1
+PKG_LICENSE:=GPL-3.0
```

```suggestion
PKG_LICENSE:=GPL-3.0
PKG_LICENSE_FILES:=LICENSE
PKG_MAINTAINER:=Daniel Golle <daniel@makrotopia.org>
```

and put it below PKG_SOURCE stuffs.

BTW this is a little bit confusing. [i2csfp.c](https://github.com/ericwoud/i2csfp/blob/5d85c6cd4c032ce6b78688c84976f65d475a1725/i2csfp.c) declears it's licensed under GPL-2.0, while this repo uses GPL-3.0.